main.nf

```diff
@@ -344,15 +420,15 @@ if (params.bam) {
     maxRetries 1
 
     input:
-    ksize
-    molecule
-    log2_sketch_size
+    each ksize from ksizes
```

Comment:
this was intentional! I don't wanna compute and process all ksizes, molecule types for all permutations for every bam. I want to do it once on the first molecule and just do sourmash compute on the fastas formed later, please change it back!

Reply:
we could switch it to extract fastas when the bam2fasta package exists. this is a workaround for now

Reply:
it is actually doing sourmash compute on a bam file for the 1st permutation of ksize, molecule, and log2_sketch_size, I think there is a comment in reference to that, sorry if I didn't add it